lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "lerna": "2.10.2",
-  "version": "2.0.0-alpha.7",
+  "version": "2.0.1",
```

Comment:
@webmaster128 , should `version` be `2.0.2` given the mess that we've created?  I'm not familiar with lerna.

Reply:
Yes. Here you find all the info how to make the release (directly from master, no PR):

https://github.com/iov-one/iov-core/wiki/Publishing

Reply:
`./node_modules/.bin/lerna publish` tags, pushes and publishes on npm. So if you did that, you're done and having a PR is pointless.

Reply:
The add nonces step 3. ensures that all packages have a change (creation of an empty file), which ensures that all packages get the same version. If you don't do that, only packages with changes get a new version.

Reply:
Thank-you, Simon.